client/settings/dpcs-settingspanel/settingspanel.py

```diff
@@ -0,0 +1,143 @@
+#coding: utf-8
+
+import json
+import gi
+gi.require_version('Gtk', '3.0')
+from gi.repository import Gtk
+import os.path
+
+# the window title
+PANELTITLE = "DPCS Settings"
+
+# about DPCS - authors, basic info, licensing and such...
+ABOUT_DPCS = """ To be written!
+"""
+
+# minimal window width and height
+WIDTH = 350
+HEIGHT = 400
+# settings dictionary
+settings = {}
+#config file path
+FILE = '/etc/dpcs.conf'
+TEMPFILE = 'dpcs.conf'
```

Comment:
The temporary file should be created under `/tmp` directory.

Is it possible to avoid creating one?